### PR TITLE
[Documentation for in Editor Training]

### PR DESCRIPTION
**Pull request by vincentpierre**
__Do not merge until `develop-communicator-redesign`__

### CODE REVIEW

File: docs/Learning-Environment-Executable.md [hunk 1]
```diff
@@ -0,0 +1,75 @@
+# Unsing an Environment Executable
+
+This section will help you create and use built environments rather than the editor to interact with an environment. Using an executable has some advantages over using the editor : 
+
+ * You can exchange executable with other peoples without having to share your entire repository.
```
Comment (awjuliani) [inline docs/Learning-Environment-Executable.md]: typo: With other *people*

File: docs/Basic-Guide.md [hunk 2]
```diff
@@ -134,6 +86,7 @@ If the learn.py runs correctly and starts training, you should see something lik
 ![Training running](images/training-running.png)
 
 You can press Ctrl+C to stop the training, and your trained model will be at `ml-agents/python/models/<run-identifier>/<env_name>_<run-identifier>.bytes`, which corresponds to your model's latest checkpoint. You can now embed this trained model into your internal brain by following the steps below, which is similar to the steps described [above](#play-an-example-environment-using-pretrained-model).
```
Comment (xiaomaogy) [inline docs/Basic-Guide.md]: After your change of in-editor traning, the trained model is no longer at this place, also it is no longer in this format.

File: docs/Basic-Guide.md [hunk 3]
```diff
@@ -41,22 +41,7 @@ In order to use ML-Agents within Unity, you need to change some Unity settings f
...
-`Assets/ML-Agents/Examples/3DBall/`.
-5. Double-click the `3DBall` file to load the scene containing the Balance 
-Ball environment.
-
-![3DBall Scene](images/mlagents-Open3DBall.png)
-
+## Setting the Brain to External
 Since we are going to build this environment to conduct training, we need to 
 set the brain used by the agents to **External**. This allows the agents to 
 communicate with the external training process when making their decisions.
```
Comment (xiaomaogy) [inline docs/Basic-Guide.md]: Now that the in-editor training is no longer needing a build of executable, We should incorporate this section as a subsection into "Training the Brain with Reinforcement Learning"

File: docs/Basic-Guide.md (hunk 2 above)
Comment (xiaomaogy) [reply docs/Basic-Guide.md]: Sorry, just saw the note section after this. This part for the training after the build of executable should be in the "Learning-Environment-Executable.md" page.

File: docs/Learning-Environment-Executable.md [hunk 4]
```diff
@@ -0,0 +1,75 @@
...
+
+```python
+from unityagents import UnityEnvironment
+env = UnityEnvironment(file_name=<env_name>)
+```
+
+If you are using `learn.py` you can pass the name of the executable in the command line. For instance : 
+
+```
+python3 learn.py <env_name> --run-id=<run-identifier> --train
```
Comment (xiaomaogy) [inline docs/Learning-Environment-Executable.md]: This section should be extended into an individual section named something like "Training the Brain with Reinforcement Learning", with part of the content in the Basic Guide moved here.

File: docs/Learning-Environment-Executable.md [hunk 5]
```diff
@@ -0,0 +1,75 @@
+# Using an Environment Executable
+
+This section will help you create and use built environments rather than the Editor to interact with an environment. Using an executable has some advantages over using the Editor : 
+
+ * You can exchange executable with other people without having to share your entire repository.
+ * You can put your executable on a remote machine for faster training.
+ * You can use `Headles` mode for faster training.
```
Comment (xiaomaogy) [inline docs/Learning-Environment-Executable.md]: Headles -> Headless

**Pull request merged**